### ISSUE TITLE
automatically create a schema does not work

### ISSUE THREAD

**Issue by aprilahijriyan** [vitalik/django-ninja#190]
I don't know why this happened, because before it worked normally. 😕 

Log:

```
Internal Server Error: /api/openapi.json
Traceback (most recent call last):
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/views.py", line 25, in openapi_json
    schema = api.get_openapi_schema()
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/main.py", line 339, in get_openapi_schema
    return get_schema(api=self, path_prefix=path_prefix)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 22, in get_schema
    openapi = OpenAPISchema(api, path_prefix)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 44, in __init__
    ("paths", self.get_paths()),
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 56, in get_paths
    path_methods = self.methods(path_view.operations)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 67, in methods
    result[method.lower()] = self.operation_details(op)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 81, in operation_details
    "responses": self.responses(operation),
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 186, in responses
    schema, _ = self._create_schema_from_model(model, by_alias=False)
  File "/home/titiw/.local/share/virtualenvs/superapp-JbhZlcYY/lib/python3.9/site-packages/ninja/openapi/schema.py", line 135, in _create_schema_from_model
    schema = model_schema(model, ref_prefix=REF_PREFIX, by_alias=by_alias)
  File "pydantic/schema.py", line 167, in pydantic.schema.model_schema
  File "pydantic/schema.py", line 548, in pydantic.schema.model_process_schema
  File "pydantic/schema.py", line 589, in pydantic.schema.model_type_schema
  File "pydantic/schema.py", line 241, in pydantic.schema.field_schema
  File "pydantic/schema.py", line 495, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 842, in pydantic.schema.field_singleton_schema
  File "pydantic/schema.py", line 548, in pydantic.schema.model_process_schema
  File "pydantic/schema.py", line 589, in pydantic.schema.model_type_schema
  File "pydantic/schema.py", line 241, in pydantic.schema.field_schema
  File "pydantic/schema.py", line 440, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 773, in pydantic.schema.field_singleton_schema
  File "pydantic/schema.py", line 667, in pydantic.schema.field_singleton_sub_fields_schema
  File "pydantic/schema.py", line 495, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 842, in pydantic.schema.field_singleton_schema
  File "pydantic/schema.py", line 548, in pydantic.schema.model_process_schema
  File "pydantic/schema.py", line 589, in pydantic.schema.model_type_schema
  File "pydantic/schema.py", line 241, in pydantic.schema.field_schema
  File "pydantic/schema.py", line 440, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 773, in pydantic.schema.field_singleton_schema
  File "pydantic/schema.py", line 667, in pydantic.schema.field_singleton_sub_fields_schema
  File "pydantic/schema.py", line 495, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 842, in pydantic.schema.field_singleton_schema
  File "pydantic/schema.py", line 548, in pydantic.schema.model_process_schema
  File "pydantic/schema.py", line 589, in pydantic.schema.model_type_schema
  File "pydantic/schema.py", line 241, in pydantic.schema.field_schema
  File "pydantic/schema.py", line 495, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 840, in pydantic.schema.field_singleton_schema
KeyError: <class 'pydantic.main.Provider'>
```

**Comment by vitalik** [comment]
Hi 
@aprilahijriyan 

can you provide your ninja version , django version and pydantic version ?

**Comment by aprilahijriyan** [comment]
Thanks for the fast response. Here's the version of the package I'm using.

```
$ pip show django django-ninja pydantic
Name: Django
Version: 3.2.6
Summary: A high-level Python Web framework that encourages rapid development and clean, pragmatic design.
Home-page: https://www.djangoproject.com/
Author: Django Software Foundation
Author-email: foundation@djangoproject.com
License: BSD-3-Clause
Requires: asgiref, pytz, sqlparse
Required-by: django-rq, django-redis, django-ninja, django-extensions, django-cors-headers
---
Name: django-ninja
Version: 0.13.2
Summary: Django Ninja - Fast Django REST framework
Home-page: https://django-ninja.rest-framework.com
Author: Vitaliy Kucheryaviy
Author-email: ppr.vitaly@gmail.com
License: None
Requires: Django, pydantic
Required-by: 
---
Name: pydantic
Version: 1.8.2
Summary: Data validation and settings management using python 3.6 type hinting
Home-page: https://github.com/samuelcolvin/pydantic
Author: Samuel Colvin
Author-email: s@muelcolvin.com
License: MIT
Requires: typing-extensions
Required-by: django-ninja
```

**Comment by stephenrauch** [comment]
I think this is a duplicate of #214

**Comment by vitalik** [comment]
@aprilahijriyan 
this will better described in next release 

for now follow #214
